apps/judicial-system/api/src/app/modules/case-list/models/caseList.model.ts

```diff
@@ -104,4 +105,7 @@ export class CaseListEntry {
 
   @Field({ nullable: true })
   readonly postponedIndefinitelyExplanation?: string
+
```

Comment:
Please consider removing the new field from the case list model as the field is not used in case lisets and is not returned from the backend.